source/common/http/conn_manager_impl.cc

```diff
@@ -696,6 +702,11 @@ ConnectionManagerImpl::ActiveStream::ActiveStream(ConnectionManagerImpl& connect
   }
 }
 
+ConnectionManagerImpl::ActiveStream::~ActiveStream() {
+  TRACE_EVENT_END("core",
```

Comment:
Would it be better to stick to RAII examples if this can't be usefully consumed by visualization tooling?

Reply:
Right. I dropped the cross-function example.